Corgibytes.Freshli.Cli/Functionality/Git/GitCommit.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Corgibytes.Freshli.Cli.Functionality.Git;
+
+public class GitCommit
+{
+    public readonly DateTimeOffset CommittedAt;
+    public readonly string ShaIdentifier;
```

Comment:
Love it or hate it, the convention I've seen throughout C# is to favor get-only properties over public readonly member variables.

```suggestion
    public DateTimeOffset CommittedAt { get; }
    public string ShaIdentifier { get; } 
```